share/functions/__fish_git_prompt.fish

```diff
@@ -416,10 +421,20 @@ function __fish_git_prompt --description "Prompt function for Git"
     if test -n "$u"
         set u "$___fish_git_prompt_color_untrackedfiles$u$___fish_git_prompt_color_untrackedfiles_done"
     end
+
     set b (string replace refs/heads/ '' -- $b)
+    set -q __fish_git_prompt_shorten_branch_char_suffix
+    or set -l __fish_git_prompt_shorten_branch_char_suffix "…"
+    if string match -qr '^\d+$' "$__fish_git_prompt_shorten_branch_len"; and test (string length "$b") -gt $__fish_git_prompt_shorten_branch_len
+        set b (string sub -l "$__fish_git_prompt_shorten_branch_len" "$b")"$__fish_git_prompt_shorten_branch_char_suffix"
+    end
     if test -n "$b"
         set b "$branch_color$b$branch_done"
     end
+    if test -n "$b"
```

Comment:
I am admittedly a bit sleepy, but why is this duplicated?

Reply:
My mistake... sorry for that ! now fixed !